package org.jgs.oop.day10;

public class ExceptionDemo02 {

	public static void main(String[] args) {
		
		ExceptionDemo02 exceptionDemo02 = new ExceptionDemo02();
		int result =exceptionDemo02.test();
		System.out.println(result);

	}
	
	
	public int  test() {
		int a = 3;
		try {
			System.out.println("hello");
			//a=10;
			int b = 2/0;
			a=15;
			return  a;
			//System.out.println("hello");
		} catch (Exception e) {
			e.printStackTrace();
		   return a;
		}finally {
			a = 10;
			System.out.println("world");
		}
		
		//return a;
	}

}
